sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientInterface.scala

```diff
@@ -109,6 +109,9 @@ private[hive] trait ClientInterface {
   /** Returns the name of the active database. */
   def currentDatabase: String
```

Comment:
Yeah. I think we don't need to address database support for all catalogs in this PR?